src/swap/components/SwapProvider.tsx

```diff
@@ -46,7 +46,9 @@ export function SwapProvider({
   const { address } = useAccount();
   // Feature flags
   const { useAggregator } = experimental;
-
+  const [maxSlippage, _setMaxSlippage] = useState(
+    experimental.maxSlippage || 3,
+  );
```

Comment:
don't want this in state do we?  If the experimental prop changes, should that update here?

Reply:
The experimental prop won't change after the component is rendered. 

It will set the initial `maxSlippage` value. Any updates will be handled by `LifeCycleStatus` slippageChange maxSlippage property

https://github.com/coinbase/onchainkit/pull/1210/files#diff-9bca97e90be8d6cfaa2c6813dbf1d684ed792b80b395f6b033d3d06963f23371R91-R94

Reply:
And we update this value using `setLifeCycleStatus`'

https://github.com/coinbase/onchainkit/pull/1210/files#diff-704c4aa565af1d2becb4f70ce49ac884069cdb8e451c6e3afe603a6d47de07bcR35-R38

Reply:
Some context on why we are setting it this way 

https://github.com/coinbase/onchainkit/pull/1210#discussion_r1744323560

Reply:
If we are only using this const to initialize lifecycle status, can it just be a const?  or alternatively we could move the initiallifecycle status into a const?

Reply:
> The experimental prop won't change after the component is rendered.

@cpcramer that sentence it's incorrect! We should always expect a prop can change, that's the all point of prop. :)